libs/libc/stream/lib_blkoutstream.c

```diff
@@ -153,15 +153,15 @@ static ssize_t blkoutstream_puts(FAR struct lib_sostream_s *self,
 
       if (offset > 0)
         {
-          size_t copyin = offset + remain > sectorsize ?
```

Comment:
could we keep the name copyin? There are similar naming in other kernel:
https://github.com/search?q=repo%3Atorvalds%2Flinux+copyin&type=code&p=1